link-grammar/linkage/score.c

```diff
@@ -68,10 +69,54 @@ static float compute_disjunct_cost(Linkage lkg)
 	return lcost;
 }
 
+/// Compute the extra cost of a multi-connector being used multiple
+/// times. Any cost on that multi-connector needs to be added for each
+/// usage of it.
+static float compute_multi_cost(Linkage lkg)
+{
+	size_t lword = 0;
+	float mcost = 0.0;
+	for (size_t i = 0; i < lkg->num_words; i++)
+	{
+		Disjunct * dj = lkg->chosen_disjuncts[i];
+		if (NULL == dj) continue;
+
+		// Look at the right-going connectors.
+		for (Connector* rcon = dj->right; rcon; rcon=rcon->next)
```

Comment:
Why the `left` connectors are ignored?

Reply:
Because this was a prototype and I did not do those.